### DISCUSSION TITLE
Remove Dummy Text ('Discover My...') from blog page ?

### DISCUSSION THREAD

**Discussion by henkvantijen** [GetPublii/Publii#1699] [Help] [answered]
I've made a site with a static home page and a separate page for the posts. Template:Simple3.0.0... So far it seems to be working nicely. 
But there's dummy text above the posts listing, and I cannot find where I should be able to remove/alter it.  The text is the same as on the screenshots of the template and starts with _Discover My Blogging Journey: Adventures, Travels, and Hobbies!.._ 

Where is it?

**Comment by michaelfluegge** [comment, marked as answer]
Try Theme > Layout > Homepage > Hero Text

**Reply by henkvantijen** [reply]
Thx.  I'm a total newbie and for me the 'blog page'  is an auxiliary page, added to all static pages (working with v46) . So was not searching the Home Page attributes, buI understand where that comes from. Thx again.
( Maybe Home Page should be renamed to Index Page? )
